src/domain/eventSearch/utils.tsx

```diff
@@ -443,3 +446,10 @@ export const getSearchQuery = (filters: Filters): string => {
 
   return buildQueryFromObject(newFilters);
 };
+
+export const getEventTypeFromRouteUrl = (url: string): EventType => {
+  if (matchPath(url, { path: ROUTES.COURSES, exact: true, strict: false })) {
+    return 'course';
+  }
+  return 'event';
+};
```

Comment:
Maybe object map and looping could be used here. Would be easier to 'all' in the future. for example:

```
const routeToEventType = {
  [ROUTES.COURSES]: 'coure;
  [ROUTES.EVENTS]: 'event';
};
```

Reply:
Added some changes.